tests/Makefile

```diff
@@ -114,7 +114,7 @@ endif
 endif
 
 check: $(BINARIES)
-	perl scripts/run-test-suites.pl
+	perl scripts/run-test-suites.pl -d
```

Comment:
I'm not so keen on making this the default behaviour, and if it must be, I'm troubled by the idea that the output is the second run. We don't have the consistent deterministic behaviour in our tests we should have, and this could be confusing, if a test fails, then passes on the second run.

A casual user wants to know if it passes tests or not - not debug information. If this is the default behaviour for `make check` why are we bothering to pass options to `run-test-suites.pl`? It should be its default behaviour too.